temoa/utilities/migrate_v4_to_v4_1.py

```
@@ -0,0 +1,313 @@
+    except Exception:
+        if temp_path.exists():
+            os.remove(temp_path)
+        raise
+    finally:
+        con_old.close()
+        con_new.close()
+
+
+def migrate_sql_dump(source_path: Path, schema_path: Path, output_path: Path) -> None:
+    if not source_path.is_file():
+        raise FileNotFoundError(f'Input SQL dump not found: {source_path}')
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)